packages/main/src/Link.js

```diff
@@ -279,11 +280,31 @@ class Link extends UI5Element {
 	}
 
 	_onkeydown(event) {
+		if (isEnter(event)) {
+			const executeEvent = this.fireEvent("click", null, true);
+
+			if (executeEvent) {
+				this.href && window.open(this.href, this.target);
```

Comment:
If I use ENTER and click a link with "href", for example this one at the top within the Link test page http://localhost:8080/test-resources/pages/Link.html


<img width="209" alt="Screenshot 2021-06-11 at 17 26 43" src="https://user-images.githubusercontent.com/15702139/121702149-4a1c9300-cada-11eb-818b-b0ed650d9ee5.png">